components/multi-platform-controller/production-downstream/stone-prod-p02/host-values.yaml

```diff
@@ -338,30 +338,44 @@ staticHosts:
     secret: ibm-s390x-ssh-key
     user: root
 
-  s390x-static-13:
-    address: 10.130.79.9
+  s390x-static-x12:
+    address: 10.130.103.8
     concurrency: '4'
     platform: linux/s390x
-    secret: old-ibm-s390x-ssh-key
+    secret: ibm-s390x-ssh-key
     user: root
 
-  s390x-static-14:
-    address: 10.130.79.10
+  s390x-static-x13:
+    address: 10.130.103.40
     concurrency: '4'
     platform: linux/s390x
-    secret: old-ibm-s390x-ssh-key
+    secret: ibm-s390x-ssh-key
     user: root
 
-  s390x-static-15:
-    address: 10.130.79.38
+  s390x-static-x14:
+    address: 10.130.103.72
     concurrency: '4'
     platform: linux/s390x
-    secret: old-ibm-s390x-ssh-key
+    secret: ibm-s390x-ssh-key
     user: root
 
-  s390x-static-16:
-    address: 10.130.79.39
+  s390x-static-x15:
+    address: 10.130.103.9
     concurrency: '4'
     platform: linux/s390x
-    secret: old-ibm-s390x-ssh-key
+    secret: ibm-s390x-ssh-key
     user: root
+
+  s390x-static-x16:
+    address: 10.130.103.41
+    concurrency: '4'
+    platform: linux/s390x
+    secret: ibm-s390x-ssh-key
+    user: root
+
+  s390x-static-x17:
+    address: 10.130.103.73
+    concurrency: '4'
+    platform: linux/s390x
+    secret: ibm-s390x-ssh-key
+    user: root
```

Comment:
missing empty line and you need to regerate kueue config since this change is adding capacity to s390x